src/modules/datepicker.js

```diff
@@ -4,7 +4,9 @@ import "./datepicker/js/bootstrap-datepicker.js";
 import "./datepicker/css/datepicker.css";
 
 const today = new Date().getTime();
-const startDay = new Date("01/01/2018").getTime();
+const lastMonth = new Date(today);
+lastMonth.setMonth(lastMonth.getMonth()-1)
```

Comment:
I might suggest using a `let` declaration here instead of `const`. The way I read the code, a `const` declaration implies to me that the variable will not be changed, whereas, `let` is inherently more flexible:
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/